strongbox-parent/pom.xml

```diff
@@ -59,7 +59,8 @@
         <version.carlspring.commons.io>1.0-SNAPSHOT</version.carlspring.commons.io>
         <version.carlspring.commons.http>1.0-SNAPSHOT</version.carlspring.commons.http>
         <version.carlspring.logback.config>1.0-SNAPSHOT</version.carlspring.logback.config>
-
+        <version.carlspring.npm.metadata>1.0.0-SNAPSHOT</version.carlspring.npm.metadata>
```

Comment:
I think a version of `1.0` would be sufficient, as all our other modules are like that.

Reply:
changed to `1.0-SNAPSHOT`